CONVENTIONS.md

```diff
@@ -0,0 +1,8 @@
+# Bitflags
+
+We represent sets of constants whose values are mutually exclusive on a bit
+level -- that is, for all flags `A` and `B` of the set with `A != B` holds
+`A & B = 0` -- by types defined using the `bitflags!` macro from the
```

Comment:
there's a little deviation from this rule for things like `S_IWRXU` and friends in the `mode` parameter to `open(2)`, `creat(2)` and some others around the libraries I think.